src/common/diffPositionMapping.ts

```diff
@@ -91,9 +90,9 @@ export function getLastDiffLine(prPatch: string): DiffLine {
 export function getDiffLineByPosition(diffHunks: DiffHunk[], diffLineNumber: number): DiffLine {
 	for (let i = 0; i < diffHunks.length; i++) {
 		let diffHunk = diffHunks[i];
-		for (let i = 0; i < diffHunk.diffLines.length; i++) {
-			if (diffHunk.diffLines[i].positionInHunk === diffLineNumber) {
-				return diffHunk.diffLines[i];
+		for (let j = 0; j < diffHunk.diffLines.length; j++) {
```

Comment:
This is a test when clicking the + sign on the line of code that has an existing comment from GitHub.com